foodalert/static/foodalert/js/host.js

```diff
@@ -21,10 +22,12 @@ Object.assign(window.vueData.components, {
     'ended-template': EndedTemplate,
     'ended-controller': EndedController,
     'welcome-template': WelcomeTemplate,
+    'close-template': CloseTemplate,
 });
 
 let rootPath = window.vueData.routes.find(obj => obj.path === "/");
 rootPath.children.find(obj => obj.path === "update").component = UpdateController;
 rootPath.children.find(obj => obj.path === "ended").component = EndedController;
 rootPath.children.find(obj => obj.path === "welcome").component = WelcomeTemplate;
 rootPath.children.find(obj => obj.path === "").component = FormController;
+rootPath.children.find(obj => obj.path === "close").component = CloseTemplate;
```

Comment:
This can not required as the default value we set for /close is already CloseTemplate. See base.js line 36.
The same goes for /welcome